packages/botonic-plugin-flow-builder/README.md

```diff
@@ -1 +1,104 @@
+<p align="center">
+    <img alt="Flow Builder button" title="flow-builder-button" src="https://i.ibb.co/ZJ2fMQN/Pasted-image-20230427115219.png" width="250">
+</p>
+
+### Install the Plugin
+
+1. Install the plugin in your bot's project by running:
+
+```bash
+npm install @botonic/plugin-flow-builder
+```
+
+## Use
+
+### Configuring the bot
+
+1. Add the following to you bot's `plugins.js` file:
+```ts
+import * as hubtypeFlowBuilder from '@botonic/plugin-flow-builder'
+
+const flowBuilderOptions = {
+  flowUrl: 'HUBTYPE_FLOW_BUILDER_URL',
+  getAccessToken: () => 'HUBTYPE_FLOW_BUILDER_ACCESS_TOKEN',
+  getLocale: () => 'YOUR_LOCALE',
+}
+
+export const plugins = [
+  {
+    id: 'hubtypeFlowBuilder',
+    resolve: hubtypeFlowBuilder,
+    options: flowBuilderOptions,
+  },
+]
+
+```
+By doing this, we are passing configuration parameters to the Plugin. 
+Con esto, le pasamos las options, los parámetros de configuración, al bot. De esta manera le podemos pasar pámetros al Plugin desde fuera. Esto nos da más flexibilidad a la hora de configurar el plugin. 
+
+Below are the parameters that we can pass to the plugin:
+-   `flowUrl`: This is the URL where the Flow is located. The bot will automatically collect this URL, so in most cases, we don't have to pass it. It is used in cases where the Flow URL is in a testing environment.
+-   `flow`: In some situations, we may want to test a flow locally instead of using the Flow Builder service. To do this, we can define the flow and its corresponding tree of nodes in a JSON file and pass it to our plugin through this variable. By doing so, we can run and test our flow locally without relying on the external service.
+-   `customFunctions`: We are able to pass custom functions to the plugin by defining them in our code and then passing them as parameters. This allows us to extend the functionality of the plugin beyond its default capabilities and execute custom logic that is tailored to our specific needs. We can add custom functions in the frontend of the Flow Builder and pass them to the plugin through this variable.
+-   `getLocale`: We can pass an array of locales to the plugin to specify which language our bot will use. 
+-   `getAccessToken`: When our bot is deployed in Hubtype, the plugin will automatically retrieve the access token. However, when testing our bot locally, we need to pass the access token as a variable.
+    
+    To retrieve the access token needed for testing our bot locally, we can follow these steps:
+	* In the console, search for "flow_builder_token" inside of "Network".
+	* Click on "preview".
+	* Here we can see the object that contains the access token.
+<p align="center">
+    <img alt="Access token" title="access-token" src=" https://i.ibb.co/9V2kSd3/Pasted-image-20230427120910.png" width="650">
+</p>
```

Comment:
this should not be part of this documentation. I think it will be better to talk with sb of the team to abilitate a public endpoint to obtain it.

Reply:
Copy! Deleted from the doc